internal/pkg/cli/env_init.go

```diff
@@ -371,7 +371,7 @@ func (o *initEnvOpts) validateCustomizedResources() error {
 	if (o.importVPC.isSet() || o.adjustVPC.isSet()) && o.defaultConfig {
 		return fmt.Errorf("cannot import or configure vpc if --%s is set", defaultConfigFlag)
 	}
-	if o.internalALBSubnets != nil && !o.importVPC.isSet() {
+	if o.internalALBSubnets != nil && (o.adjustVPC.isSet() || o.defaultConfig) {
```

Comment:
No-- there are tests to make sure the `--internal-alb-subnets` flag _isn't_ used with `--default-config` or the `--override` flags, but when I mistakenly accepted the change that made the validation more restrictive (not allowing `--internal-alb-subnets` unless `--import` flags were used), I didn't add tests because I didn't realize that the change was bigger than just streamlining the logic.

Reply:
Oh that was my bad. I thought internal ALB can't be used unless they import VPC

Reply:
But right they could do that without the flag for sure.